app/stores/AdaRedemptionStore.js

```diff
@@ -30,6 +30,7 @@ export default class AdaRedemptionStore extends Store {
   @observable amountRedeemed: number = 0;
   @observable showAdaRedemptionSuccessMessage: boolean = false;
   @observable redeemAdaRequest: Request<Wallet> = new Request(this.api.redeemAda);
+  @observable redeemAdaSubmitted: boolean = false;
```

Comment:
I would rename this to `adaSuccessfullyRedeemed` because it's not only about the "submit" state (which you could ask for via `redeemAdaRequest.wasExecuted` but about success state.